### PR TITLE
Add $BEAM (Beam) Token To prices model

### PR DESCRIPTION
**Pull request by 0x0sub**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by 0x0sub** [comment]
recheck

**Comment by 0x0sub** [comment]
I have read the CLA Document and I hereby sign the CLA

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1623,7 +1623,8 @@ FROM
     ('mav-maverick-token', 'ethereum', 'MAV', 0x7448c7456a97769f6cd04f1e83a4a23ccdc46abd, 18),
     ('pepe-pepe', 'ethereum', 'PEPE', 0x6982508145454Ce325dDbE47a25d4ec3d2311933, 18),
     ('wld-worldcoin', 'ethereum', 'WLD', 0x163f8c2467924be0ae7b5347228cabf260318753, 18),
-    ('val-valeria', 'ethereum', 'VAL', 0x011e128ec62840186f4a07e85e3ace28858c5606, 18)
+    ('val-valeria', 'ethereum', 'VAL', 0x011e128ec62840186f4a07e85e3ace28858c5606, 18),
+    ('beam-beam-eth', 'ethereum', 'BEAM', 0x62d0a8458ed7719fdaf978fe5929c6d342b0bfce, 18)
```
Comment (Hosuke) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1627]: API: https://api.coinpaprika.com/v1/coins/beam-beam-eth

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @0x0sub

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**